# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../firefox/MOZ_PLUGIN_PATH.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./xpfe/bootstrap/mozilla.in.vanilla	2005-09-21 12:45:56.000000000 +0200
+++ ./xpfe/bootstrap/mozilla.in	2005-09-21 12:48:39.000000000 +0200
@@ -119,6 +119,17 @@
   fi
 fi
 
+#
+## Make sure that we set the plugin path for backwards compatibility
+##
+if [ "$MOZ_PLUGIN_PATH" ]
+then
+  MOZ_PLUGIN_PATH=$MOZ_PLUGIN_PATH:/usr/lib/mozilla/plugins
+else
+  MOZ_PLUGIN_PATH=/usr/lib/mozilla/plugins
+fi
+export MOZ_PLUGIN_PATH
+
 script_args=""
 moreargs=""
 debugging=0
